src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/product.html.twig

```diff
@@ -52,6 +52,28 @@
   </li>
 {% endblock %}
 
+{% block packed_product_row %}
+  <li class="packed-product entity-item" id="{{ form.vars.name }}">
+    <div class="packed-product-image">
+      {{ form_widget(form.image) }}
+    </div>
+    <div class="packed-product-legend">
+      {{ form_widget(form.name, {prefix: '<i class="material-icons delete entity-item-delete">delete</i>'}) }}
```

Comment:
```suggestion
      {{ form_widget(form.name, {prefix: '<i class="material-icons entity-item-delete">delete</i>'}) }}
```

Is that class used somewhere?

Reply:
You're right it's not @Amoifr this one can be removed

Reply:
Yes, I also done it for related product that has been build in the same way